config/routes/overrides.rb

```diff
@@ -1,6 +1,21 @@
+class WhitelistConstraint
+  def initialize
+    @path = /^\/api\/v2\/organizations\/\S+\/parameters/
```

Comment:
If this intended to potentially hold multiple routes, should this be `@paths` and an array?